examples/poyo/train.py

```
@@ -148,48 +148,51 @@ def __init__(self, cfg: DictConfig):
-        )
-        self.test_dataset.disable_data_leakage_check()
+        example_recording = train_ds.get_recording(train_ds.recording_ids[0])
+        readout_id = example_recording.config["readout"]["readout_id"]
+        for recording_id in train_ds.recording_ids:
+            recording = train_ds.get_recording(recording_id)
+            if readout_id != recording.config["readout"]["readout_id"]:
+                raise ValueError(
+                    f"Readout ID mismatch: expected '{readout_id}' but got "
+                    f"'{recording.config['readout']['readout_id']}' for recording '{recording_id}'."
+                    f"POYO only supports a single readout"
+                )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)